tools/performance/PowerShell.stacktags

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Tag Name="PowerShell">
+  <Tag Name="Pipeline Execution">
+    <Tag Name="Interpreter">
+      <Entrypoint Module="System.Management.Automation.dll" Method="System.Management.Automation.Interpreter.*"/>
+      <Entrypoint Module="System.Core.dll" Method="System.Dynamic.UpdateDelegates.*"/>
+      <Entrypoint Module="System.Core.dll" Method="System.Runtime.CompilerServices.CallSiteBinder.*"/>
+      <Entrypoint Module="System.Management.Automation.ni.dll" Method="System.Management.Automation.Interpreter.*"/>
+      <Entrypoint Module="System.Core.ni.dll" Method="System.Dynamic.UpdateDelegates.*"/>
+      <Entrypoint Module="System.Core.ni.dll" Method="System.Runtime.CompilerServices.CallSiteBinder.*"/>
+    </Tag>
+    <Tag Name="CommandDiscovery">
+      <Entrypoint Module="System.Management.Automation.dll" Method="System.Management.Automation.CommandDiscovery.*"/>
```

Comment:
Could you add a comment why sometimes the module name has `.ni` while sometimes it doesn't

Reply:
Uh, I'm not sure, and maybe not relevant to PowerShell Core. This file isn't really that refined at all, so the hope is folks will contribute and make it more useful,

Reply:
I remember now, comment added.